lib/puppet/util/FFI/ffi_constants.rb

```diff
@@ -0,0 +1,302 @@
+require 'puppet/util/ffi'
+require 'ffi'
+
+module Puppet::Util::FFI
+  module FFIConstants
```

Comment:
It looks like this file was added on a case-sensitive filesystem, but some aren't. I'd expect this file to be `lib/puppet/util/ffi/ffi_constants.rb`.